### PR TITLE
Bump version to 8.0.0

### PR DESCRIPTION
**Pull request by PululuK**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop 
| Description?      | Update version to 8.0.0
| Type?             | improvement 
| Category?         | CO 
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | ~
| How to test?      | ~
| Possible impacts? | ~

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/24172)

### CODE REVIEW

**Review by matks — DISMISSED** [review]

**Review by Progi1984 — CHANGES_REQUESTED** [review]
An internal decision is in WIP about the next version

**Comment by marionf** [comment]
Hello @PululuK 

The next version will be the V8 and it will be a major version. This is why: in an effort of innovation and technological attractiveness, the update of the techno (PHP8 and Symfony 4.4) imposes the passage to the numbering of version in 8.x.x. This is not a breakthrough version unlike passage 1.6 -> 1.7, but a version in the continuity of the work in progress on 1.7. Therefore, there will be no need for a complete overhaul to migrate. The version number changes, but the difficulty of upgrading to 1.7.8 and 8.0.0 will be as easy as upgrading from 1.7.7 to 1.7.8. We also take the opportunity to remove the 1 and respect Semver.

File: app/AppKernel.php [hunk 1]
```diff
@@ -32,10 +32,10 @@
 
 class AppKernel extends Kernel
 {
-    const VERSION = '1.7.8.0';
+    const VERSION = '1.7.9.0';
```
Comment (Progi1984) [inline app/AppKernel.php]:
```suggestion
    const VERSION = '8.0.0';
```

File: app/AppKernel.php [hunk 2]
```diff
@@ -32,10 +32,10 @@
...
     const MAJOR_VERSION_STRING = '1.7';
```
Comment (Progi1984) [inline app/AppKernel.php]:
```suggestion
    const MAJOR_VERSION_STRING = '8';
```

File: app/AppKernel.php [hunk 3]
```diff
@@ -32,10 +32,10 @@
...
     const MAJOR_VERSION = 17;
```
Comment (Progi1984) [inline app/AppKernel.php]:
```suggestion
    const MAJOR_VERSION = 8;
```

File: app/AppKernel.php [hunk 4]
```diff
@@ -32,10 +32,10 @@
...
-    const MINOR_VERSION = 8;
+    const MINOR_VERSION = 9;
```
Comment (Progi1984) [inline app/AppKernel.php]:
```suggestion
    const MINOR_VERSION = 0;
```

File: install-dev/install_version.php [hunk 5]
```diff
@@ -24,7 +24,7 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
 
-define('_PS_INSTALL_VERSION_', '1.7.8.0');
+define('_PS_INSTALL_VERSION_', '1.7.9.0');
```
Comment (Progi1984) [inline install-dev/install_version.php]:
```suggestion
define('_PS_INSTALL_VERSION_', '8.0.0');
```

**Review by Progi1984 — CHANGES_REQUESTED** [review]
Some feedbacks

**Review by Progi1984 — DISMISSED** [review]

**Comment by Progi1984** [comment]
@PululuK Thank you for your contribution. Could you rebase your PR because some files are in conflict ?

**Comment by matks** [comment]
@PululuK friendly reminder 😉

**Comment by PierreRambaud** [comment]
PR rebased and squashed

**Review by Progi1984 — DISMISSED** [review]

**Review by jolelievre — DISMISSED** [review]

**Review by NeOMakinG — DISMISSED** [review]

**Comment by PierreRambaud** [comment]
This pr needs https://github.com/PrestaShop/TranslationFiles/pull/20 :)

**Comment by PierreRambaud** [comment]
This PR also needs a change in the htaccess domain i18n.prestashop.com

**Comment by PierreRambaud** [comment]
It's in Waiting for dev, the problem comes from the PrestaShop services (i18n.prestashop.com)

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
i18n.prestashop is in error now, I'm still on it :sweat_smile:

**Review by PierreRambaud — APPROVED** [review]
Everything is ok now with the server

**Comment by PierreRambaud** [comment]
Thank you @PululuK

**Pull request merged**